### ISSUE TITLE
ECO:0000232 'chromosome conformation capture-based evidence'

### ISSUE THREAD

**Issue by rctauber** [evidenceontology/evidenceontology#156]
The label for ECO:0000232 is a bit misleading. It is a parent class for all chromosome conformation capture assays, which includes 3C, 4C, 5C, and Hi-C.

I think this label is too similar to the 3C child - ECO:0000233 (chromosome conformation capture evidence). We recently added some similar terms to OBI for the mappings, and they decided to use 'nuclear ligation assay' as the parent class for all chromosome conformation capture assays. I'm wondering if it makes sense to change the label of ECO:0000232 to be similar to that?

See: https://www.ncbi.nlm.nih.gov/pubmed/8327891

**Comment by mgiglio99** [comment]
posting James' comment here...
I think that you are right - ECO:0000232: chromosome conformation capture-based evidence and ECO:0000233: chromosome conformation capture evidence are "too similar".  

So being totally unfamiliar with this area of research, I did some reading.  From what I read, you are right on track with the idea of re-labeling ECO:0000232 as "nuclear ligation assay".

That's my $0.02 at least.

James

**Comment by mgiglio99** [comment]
Hi,
I see a bit why they went with a ligation term name - but as I'm looking at this area of the tree more, I'm wondering if that makes sense as things not under that node also have ligation in the process. So, I'm thinking we might need to look at this a bit more closely.
As a first thought and to fix the issue you describe Becky - I suggest changing the parent to "chromosome conformation evidence". And then we can rethink all of the children in here. I think there might be more to sort out in this node.
As an aside, looks like there is only one child for "experimental genomic evidence" and I'm wondering why we even have that term.
Michelle

**Comment by mgiglio99** [comment]
I was confused about which term Becky wanted to relabel - I see now its the overall parent, which makes sense. However, I'm still not too keen on that name as ligation to me is a broader thing than just these types of assays. Again, I think it might be worth spending some time with this whole node.

**Comment by rctauber** [comment]
I agree that it would be beneficial to look at the node as a whole @mgiglio99 ... perhaps do something similar as we did with the biological evidence? If we want to relabel it as 'nuclear ligation', we may have to move some things around...

Also, are you looking at ECO:0000073 (experimental genomic evidence)? I see four children.

**Comment by mgiglio99** [comment]
Hi Becky, 
I see now that there are more children. I haven't quite gotten the hang of using the ECO viewer on the website.

**Comment by jbmunro** [comment]
This branch was reviewed.  We retained the parent term 'chromosome conformation-based evidence' and for the children, we decided to use the abbreviated common term - e.g. '3C evidence' - as opposed to what we typically do, which is use the full name - e.g. 'chromosome conformation capture evidence'.

I cannot find further discussion with regard to 'nuclear ligation assay evidence'.  It is my understanding that 'chromosome conformation-based evidence' would be a child of 'nuclear ligation assay evidence', and given that we have no sibling for 'chromosome conformation-based evidence', creating a parent term 'nuclear ligation assay evidence' does not seem necessary.
